CI/beta-release-script.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash -x
+
+cat <<EOF > /tmp/version.py
```

Comment:
better put '/tmp/version.py' in a variable that will be reused in multiple places